### DISCUSSION TITLE
Problems sending email

### DISCUSSION THREAD

**Discussion by jintaokoong** [pocketbase/pocketbase#684] [Q&A] [answered]
I'm trying to configure email via smtp server, tried with sendgrid as well as mailtrap, both gave me the following error `x509: certificate signed by unknown authority.` when I tried to send a test email. Any ideas how to go about this problem? Thanks.

Edit: I'm currently running the latest build 0.7.7 in a docker container

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, if you are using something like alpine as base image maybe the `ca-certificates` package is missing.

_I'll test later the fly.io setup with mailtrap to confirm if that's the case._

**Reply by jintaokoong** [reply]
Yes, this is exactly the issue, I was using the base `scratch` image, with `ca-certificates` missing. I should've searched more on this error message before posting. Thanks for the advice.
